packages/dev/inspector-v2/test/app/index.ts

```diff
@@ -1,6 +1,5 @@
 // eslint-disable-next-line import/no-internal-modules
-import type { ArcRotateCamera, Nullable } from "core/index";
-
+import { Color3, CreateSphere, type ArcRotateCamera, type Nullable } from "core/index";

```

Comment:
Mixed imports here (might not matter for the test app), but we don't want to import from index and pull in all the side effects.